iris/docs/version_switch.js

```diff
@@ -11,7 +11,7 @@
     // nb. Also include the latest version explictly as well as under "latest",
     // so that users can get to the versioned (non-latest) URL.
 
-    'v2.2': '2.2',
+    'v2.2.1': '2.2',
```

Comment:
This is the thing that handles the little dropdown box in the top right hand corner of the iris documentation.

Now, if someone selects "v2.2" it takes them to the `v2.2.1` documentation rather than the `v2.2.0` documentation which is now out of date.

This follows the pattern we did for previous patch releases (see v1.9 and 1.8)